.github/workflows/zowe-explorer-ftp-ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       # Continue to run tests on the other systems if one fails
       fail-fast: false
       matrix:
-        node-version: [12.x, 14.x]
+        node-version: [14.x, 16.x]
```

Comment:
Thanks for updating the workflows 🙏 
We can create a separate issue for addressing any node16 specific problems in the future 😋 
For now I do believe we can continue building with node14 for the v1-lts branch 😀